client/web/src/enterprise/user/settings/auth/UserSettingsPermissionsPage.tsx

```
@@ -65,6 +65,7 @@ export const UserSettingsPermissionsPage: React.FunctionComponent<React.PropsWit
             userID: user.id,
             query: debouncedQuery,
         },
+        errorPolicy: 'all',
```

I wonder if other unrelated errors here would now NOT show 🤔

---

Actually, thinking about this again, I think the issue is that the repository resolver should return `null` for unknown repos, and then your nice rendering down in this file will still work, without having to parse errors everywhere.

---

Things get really weird with how the GQL resolver hydrates here :(

---

I do think this might still be fine though, since the resolver _is_ trying to access repositories the admin doesn't have access to. So I think that should return an error 🤔 it's not that the repo doesn't exist